src/main/java/com/google/sps/servlets/ClassServlet.java

```diff
@@ -0,0 +1,65 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*; 
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.DatastoreServiceConfig;
+
+
+/** Servlet that creates a new class Datastore.*/
+@WebServlet("/class")
+public class ClassServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      return;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    // navigate to /_ah/admin to view Datastore
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    System.setProperty(DatastoreServiceConfig.DATASTORE_EMPTY_LIST_SUPPORT, Boolean.TRUE.toString());
+
+    String className = request.getParameter("className");
+    String enter = request.getParameter("enter");
+
+    if (enter.equals("create a new class!")){
```

Comment:
What does the enter parameter signify? Could we use a value without spaces and punctuation? A simpler name might help prevent issues in the future.

Reply:
It looks like this servlet has two functions, creating a new class and adding a student to a queue? It might be better to have to different servlets for these two functions. This could be done with two different forms in the HTML.